iroha_client/tests/add_asset_should_propagate_to_another_peer.rs

```diff
@@ -103,7 +106,7 @@ mod tests {
             .iter()
             .enumerate()
             .map(|(i, key_pair)| PeerId {
-                address: format!("127.0.0.1:{}", 1338 + i),
+                address: format!("127.0.0.1:{}", 1337 + i),
```

Comment:
Not that I mind, but why change to 1337?

Reply:
Because we had default 1337, but use 1338 in tests